lang/python/python-twisted/Makefile

```diff
@@ -61,6 +61,7 @@ endef
 define Py3Package/python3-twisted/filespec
 +|$(PYTHON3_PKG_DIR)
 -|$(PYTHON3_PKG_DIR)/twisted/conch/scripts/tkconch.py
+-|$(PYTHON3_PKG_DIR)/twisted/test
```

Comment:
There are lots of `test` directories, e.g. [_threads/test](https://github.com/twisted/twisted/tree/twisted-25.5.0/src/twisted/_threads/test), [application/test](https://github.com/twisted/twisted/tree/twisted-25.5.0/src/twisted/application/test), [application/runner/test](https://github.com/twisted/twisted/tree/twisted-25.5.0/src/twisted/application/runner/test) - this one line is insufficient.